plugins/test/unit/plugins/db/test_models.py

```diff
@@ -409,6 +413,10 @@ def test_update_needed_bad_date_existing(self):
         existing_erratum.updated = 'Fri Jan  1 00:00:00 UTC 2016'
         uploaded_erratum.updated = '2016-04-01 00:00:00 UTC'
         self.assertRaises(ValueError, existing_erratum.update_needed, uploaded_erratum)
+        existing_erratum.updated = '2015-01-01'
```

Comment:
The purpose of this test `test_update_needed_bad_date_existing` is to ensure that an exception is raised if the existing erratum has a date in unknown format. I'm not sure I understand why another assertion is added here and what it is for. Can you elaborate, please?

Reply:
Good point - I'll remove and repush